srcpkgs/praat/template

```diff
@@ -13,7 +13,7 @@ license="GPL-2.0-or-later"
 homepage="http://www.praat.org/"
 changelog="http://www.fon.hum.uva.nl/praat/manual/What_s_new_.html"
```

Comment:
```suggestion
homepage="https://www.fon.hum.uva.nl/praat/"
changelog="https://www.fon.hum.uva.nl/praat/manual/What_s_new_.html"
```

Reply:
They used to mention only www.praat.org as homepage, because the redirect might change, but in https://www.fon.hum.uva.nl/praat/manual/Intro.html the mention both adresses now, so I guess it’s ok to change `homepage` to make it look more consistent. :)